### ISSUE TITLE
Linked entries/assets are output only once per entry

### ISSUE THREAD

**Issue by mrh-is** [contentful/contentful_middleman#108]
If an entry is linked twice from the same parent entry, it will only be fully resolved the first time.

### Example

Consider the following hypothetical content models:

```
Person {
  name: String
  dog: Dog
  dogToy: Toy
}

Dog {
  name: String
  toy: Toy
}

Toy {
  name: String
}
```

(This may be an unusual data model, but it's valid and should work as expected.)

When run through this gem, they will produce approximately the following for a given `Person`:

```yaml
---
id: person1
name: Owner
dog:
  id: dog1
  name: Spot
  toy:
    id: toy1
    name: Ball
dogToy:
  id: toy1
```

Notice that the second time the `Toy` is outputted, it does _not_ show its fields, just its `id`.

This violates the expectation that a programmer would be able to use `person.dogToy.name`, without having to find the `Toy`'s information elsewhere in the YAML.

**Comment by dlitvakb** [comment]
Hey @mrh-is,

This has been fixed as part of release [1.5.0](https://github.com/contentful/contentful_middleman/blob/master/CHANGELOG.md#fixed).

Please update to that version and let me know if your issue is solved.

Cheers

**Comment by mrh-is** [comment]
I guess I will once #107 is merged. :)

**Comment by mrh-is** [comment]
This is a duplicate of #99, my bad. Slightly different language & branches combined to confuse me. Thanks for the help @dlitvakb!
